reference/docs-conceptual/samples/Managing-Processes-with-Process-Cmdlets.md

```diff
@@ -25,10 +27,10 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id ProcessName
       0       0        0         16     0               0 Idle
 ```
 
-Although it is normal for cmdlets to return no data in some situations, when you specify a process
-by its ProcessId, `Get-Process` generates an error if it finds no matches, because the usual
-intent is to retrieve a known running process. If there is no process with that Id, it is likely
-that the Id is incorrect or that the process of interest has already exited:
+Although it's normal for cmdlets to return no data in some situations, when you specify a process by
+its ProcessId, `Get-Process` generates an error if it finds no matches, because the usual intent is
+to retrieve a known running process. If there is no process with that Id, it's likely that the Id is
+incorrect or that the process of interest has already exited:
```

Comment:
```suggestion
its **ProcessId**, `Get-Process` generates an error if it finds no matches, because the usual intent
is to retrieve a known running process. If there is no process with that ID, it's likely that the ID
is incorrect or that the process of interest has already exited:
```